src/slack.ts

```diff
@@ -118,7 +124,15 @@ async function send(
   for (const [step, status] of Object.entries(jobSteps)) {
     checks.push(`${stepIcon(status.outcome)} ${step}`)
   }
-  const fields = []
+
```

Comment:
I would personally move this block before the comment // add job steps, if provided